src/main/resources/swagger/rawls.yaml

```diff
@@ -295,9 +297,48 @@ paths:
              - openid
              - email
              - profile
+     put:
+       responses:
+         '200':
```

Comment:
FYI, http://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html says 200 is acceptable, but I am changing it anyway